### PR TITLE
Please add this feaure. 

### PR DESCRIPTION
**Pull request by shaifcs**

### CODE REVIEW

File: src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -9,6 +9,7 @@ public class Main
 {
     public static void main( String[] args )
     {
+        // testing comment
```
Comment (shaifulcse) [inline src/main/java/org/example/Main.java]: this does not make sense.

File: src/main/java/org/example/Main.java [hunk 2]
```diff
@@ -9,6 +9,7 @@ public class Main
...
         boolean valid = false;
         Scanner scanner = new Scanner(System.in);
         passwordChecker passwordChecker = new passwordChecker();
```
Comment (shaifulcse) [inline src/main/java/org/example/Main.java:14]: can you change the name?

File: src/main/java/org/example/Main.java [hunk 3]
```diff
@@ -9,6 +9,7 @@ public class Main
 {
     public static void main( String[] args )
     {
+        // testing comment
         boolean valid = false;
```
Comment (shaifulcse) [inline src/main/java/org/example/Main.java]: change to isValid

**Review by shaifulcse — COMMENTED** [review]
Please fix these issues.  You can also (generalized comments. )

File: src/main/java/org/example/Main.java (hunk 1 above)
Comment (shaifcs) [reply src/main/java/org/example/Main.java]: addressed.

File: src/main/java/org/example/Main.java (hunk 2 above)
Comment (shaifcs) [reply src/main/java/org/example/Main.java:14]: changed.

File: src/main/java/org/example/Main.java (hunk 3 above)
Comment (shaifcs) [reply src/main/java/org/example/Main.java]: okay. done.

**Comment by shaifcs** [comment]
I addressed all.

**Pull request merged**